### PR TITLE
CWE-327 BrokenCryptoAlgorithm recommendation to AES instead of Blowfish

### PR DESCRIPTION
**Pull request by fatenhealy**
Changing the [BrokenCryptoAlgorithm](https://help.semmle.com/wiki/display/PYTHON/Use+of+a+broken+or+weak+cryptographic+algorithm) second example recommendation to AES instead of Blowfish for Python same as:
[JS](https://help.semmle.com/wiki/display/JS/Use+of+a+broken+or+weak+cryptographic+algorithm), [Java](https://help.semmle.com/wiki/display/JAVA/Use+of+a+broken+or+risky+cryptographic+algorithm) and [C++](https://help.semmle.com/wiki/display/CCPPOBJ/Use+of+a+broken+or+risky+cryptographic+algorithm)

### CODE REVIEW

**Review by felicitymay — DISMISSED** [review]
The change LGTM - this is a good example where it might be good to move the content of the `qhelp` file into a resuable fragment so that the set of queries stay in step.

File: python/ql/src/Security/CWE-327/examples/broken_crypto.py [hunk 1]
```diff
@@ -6,7 +6,7 @@ def send_encrypted(channel, message):
     channel.send(cipher.encrypt(message)) # BAD: weak encryption
 
 
-cipher = Blowfish.new(SECRET_KEY)
+cipher = AES.new(SECRET_KEY)
```
Comment (tausbn) [inline python/ql/src/Security/CWE-327/examples/broken_crypto.py:9]: For consistency, you should change `Blowfish` to `AES` in the `import` statement in the first line of this file.

**Review by tausbn — CHANGES_REQUESTED** [review]
One minor change needed, otherwise this looks good to me. :+1:

File: python/ql/src/Security/CWE-327/examples/broken_crypto.py (hunk 1 above)
Comment (fatenhealy) [reply python/ql/src/Security/CWE-327/examples/broken_crypto.py:9]: Thanks @tausbn 👍🏼 😃

Comment (tausbn) [reply python/ql/src/Security/CWE-327/examples/broken_crypto.py:9]: I'm slightly confused. Line one appears unchanged to me? :confused:

Comment (fatenhealy) [reply python/ql/src/Security/CWE-327/examples/broken_crypto.py:9]: oops, sorry

**Review by tausbn — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**